### PR TITLE
add: `xcb-util-errors`

### PR DESCRIPTION
**Pull request by Elsie19**
For `hyprland-bin`.

### CODE REVIEW

File: packages/xcb-util-errors/xcb-util-errors.pacscript [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name="xcb-util-errors"
+gives="lib${name}"
+pkgver="1.0.1"
+pkgdesc="XCB errors library"
+homepage='https://cgit.freedesktop.org/xcb/util-errors/'
+makedepends=("libtool" "xcb-proto" "xutils-dev" "autoconf" "autogen" "pkg-config" "libxcb1-dev" "python3")
+url="https://xorg.freedesktop.org/archive/individual/lib/xcb-util-errors-${pkgver}.tar.xz"
```
Comment (oklopfer) [inline packages/xcb-util-errors/xcb-util-errors.pacscript:7]: `url="https://xorg.freedesktop.org/archive/individual/lib/${name}-${pkgver}.tar.xz"`

**Review by oklopfer — CHANGES_REQUESTED** [review]

File: packages/xcb-util-errors/xcb-util-errors.pacscript (hunk 1 above)
Comment (Elsie19) [reply packages/xcb-util-errors/xcb-util-errors.pacscript:7]: It's not gonna change anytime soon.

Comment (oklopfer) [reply packages/xcb-util-errors/xcb-util-errors.pacscript:7]: > It's not gonna change anytime soon.

its redundant code.

**Review by oklopfer — APPROVED** [review]

File: packages/xcb-util-errors/xcb-util-errors.pacscript (hunk 1 above)
Comment (Elsie19) [reply packages/xcb-util-errors/xcb-util-errors.pacscript]: > > It's not gonna change anytime soon.
> 
> its redundant code.

Your mom

**Pull request merged**